pkgs/top-level/all-packages.nix

```diff
@@ -18120,7 +18135,21 @@ in
     # udev is the same package as systemd which depends on cryptsetup
     # which depends on lvm2 again.  But we only need the libudev part
     # which does not depend on cryptsetup.
-    udev = udev.override { cryptsetup = null; };
+    udev = udev.override {
```

Comment:
Is this a separate, or the same that's passed into `dbus` above? We should give it an attribute name and refer it in both places.

Reply:
It's the same. Something like systemdMinimal?

Reply:
:+1:

Reply:
To what should udev point? systemd or systemdMinimal?